elementary/monitor/api/tests/schema.py

```diff
@@ -27,10 +27,13 @@ class DbtTestResultSchema(BaseModel):
     failed_rows_count: Optional[int] = None
 
 
-class TestMetadataSchema(BaseModel):
+class RawTestResultSchema(BaseModel):
```

Comment:
Why is it called "Raw"?

Reply:
So we have 2 different schemas for test result.
One is the rows of the table elementary test results, and one is for the actual test results object we have in the report.

Can change the naming into "TestResultRowSchema" or "DbTestResultSchema" 🙂